sdk/nodejs/runtime/settings.ts

```diff
@@ -51,7 +53,25 @@ export interface Options {
 /**
  * options are the current deployment options being used for this entire session.
  */
-const options = loadOptions();
+let options = loadOptions();
+
+
+export function setMockOptions(mockMonitor: any, project?: string, stack?: string, preview?: boolean) {
```

Comment:
Why `mockMonitor: any`?

Reply:
IIRC I ran into some sort of circular import problems if I imported the `Mocks` interface from `./runtime` or vice-versa. We could redeclare that interface here if that seems better.